x-pack/plugins/security_solution/cypress/e2e/detection_alerts/expandable_flyout/alert_details_url_sync.cy.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../../objects/rule';
+import { cleanKibana } from '../../../tasks/common';
+import { waitForAlertsToPopulate } from '../../../tasks/create_new_rule';
+import { expandFirstAlertExpandableFlyout } from '../../../tasks/document_expandable_flyout';
+import { login, visit } from '../../../tasks/login';
+import { createRule } from '../../../tasks/api_calls/rules';
+import { ALERTS_URL } from '../../../urls/navigation';
+import {
+  DOCUMENT_DETAILS_FLYOUT_CLOSE_BUTTON,
+  DOCUMENT_DETAILS_FLYOUT_HEADER_TITLE,
+} from '../../../screens/document_expandable_flyout';
+
+// Skipping these for now as the feature is protected behind a feature flag set to false by default
+// To run the tests locally, add 'securityFlyoutEnabled' in the Cypress config.ts here https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/config.ts#L50
+describe.skip('Expandable flyout state sync', { testIsolation: false }, () => {
+  const rule = getNewRule();
+
+  before(() => {
+    cleanKibana();
+    login();
+    createRule(rule);
```

Comment:
I'm going to be making a change to my tests to instead load an alert whose `id` we already have so we can test the full url param rather than just the key being present.